src/app/services/sms/sms.service.ts

```diff
@@ -212,7 +216,17 @@ const sendSms = (
     },
   )
     .andThen(({ status, sid, errorCode, errorMessage }) => {
+      const ddTags = {
+        // msgsrvcsid: msgSrvcSid,
```

Comment:
any reason why this is commented out?

Reply:
I'll add a comment for this. Tags should only contain values with limited cardinality, and I don't have a good sense of the cardinality of msgSrvcSid. I included it because I thought we might be able to see patterns of failures for a specific Message Service provider, but I reckon, it's not that urgent, or can be investigated when we see a raise of errors.

Reply:
https://github.com/opengovsg/FormSG/pull/3824/commits/2b72767f95f59beb67d66c1855a03bc0bde914a2